.github/workflows/CI.yml

```diff
@@ -160,6 +160,10 @@ jobs:
       env: 
         ADO_PAT: ${{ secrets.ADO_PAT }}
         GH_PAT: ${{ secrets.GH_PAT }}
+        GH_SOURCE_PAT: ${{ secrets.GH_SOURCE_PAT }}
```

Comment:
isn't this going to cause the GHEC -> GHEC integration test to use this as the source PAT also?

Reply:
Oops! you're right, I totally missed it. Maybe the way to go here is to name it differently like `GHES_PAT` and then pass it to the CLI as an arg in the test.

Reply:
that sounds like it would work

Reply:
That might be messy to make work. You probably need to put some code in the test to override/reset the envvars properly for the test scenario.